gradebook/test/GradeBook.tests/BookTest.cs

```diff
@@ -8,7 +8,19 @@ public class BookTest
         [Fact]
         public void Test1()
         {
+            //arrange
+            var book = new Book("GBook Name", "User 1");
+            book.AddGrade(20);
+            book.AddGrade(30);
+            book.AddGrade(40);
 
+            //act
+            var result = book.CalculateStats();
+
+            //assert
+            Assert.Equal(30, result.Avg, 1);
+            Assert.Equal(40, result.High, 1);
+            Assert.Equal(20,result.Low,1);
         }
```

Comment:
- I would add more tests for edge cases. (e.g. negative grade, grade = 0, grade = 100, etc).  You can use [`Assert.Throws`](url) for some of those.
- There's also no testing of `double`s - this would miss incorrect rounding or integer truncation.
- Precision could be less than 1 - generally this should be around the floating point delta for the range you're testing in, so I'd put this at about `0.000001` here.  You also know that `High` and `Low` should be precise (since they're returning just what you passed in), so those should have zero tolerance.
- Inconsistent spacing.